DayrSuryan/DSEthiop19.xml

```diff
@@ -273,14 +272,20 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <seg type="script">Twentieth-century bulky script.</seg>
                         <seg type="ink">Black, red.</seg>
                         <seg type="rubrication">First words of structuring units, elements of punctuation signs, numerals, the name of
-                           <foreign xml:lang="gez">ገብረ፡ ማርያም፡</foreign>, <foreign xml:lang="gez">ማርያም፡</foreign>. The word divider <foreign xml:lang="gez">፡</foreign> is often constituted of one black and one red dot,
-                              similarly, the red dot has often not been added and therefore
-                           only one dot is present (marked as . in this record). <foreign xml:lang="gez">፨</foreign> is used widely.
+                           <foreign xml:lang="gez">ገብረ፡ ማርያም፡</foreign>, <foreign xml:lang="gez">ማርያም፡</foreign>.
+                           The red and black dot <foreign xml:lang="gez">፨</foreign> is used widely; the sign
                               <foreign xml:lang="gez">፥</foreign> is used with the middle dot in red.
-                         </seg>/
+                         </seg>
                            <date>20th century.</date>
                            <desc>Irregular, crude; by a poorly trained scribe.
-                             Most probably the same hand as <ref type="hand" corresp="DSEthiop18#h1">Hand 1 of DS Ethiop. 18</ref>. </desc>
+                             The scribe executed the word divider <foreign xml:lang="gez">፡</foreign>
+                             in such a peculiar way that he interpreted as dot
+                             the last letter element protruding to the right,
+                             (cp., e.g., such letters as ኒ, ዱ, ተ, ኪ, but also ም), and
```

Comment:
cp., e.g., I find a bit much :) "such" to me means already that they are a selection, so maybe e.g. can go?

Reply:
ok

Reply:
I'll change it in the file